.storybook/themes/news.scss

```diff
@@ -3903,7 +3903,10 @@
 						),
 						"media-item": (
 							"float": left,
-							"margin": 0,
+							"margin-block-end": 0,
+							"margin-block-start": 0,
+							"margin-inline-end": 0,
+							"margin-inline-start": 0,
 							"max-width": 33%,
```

Comment:
This should be `max-inline-size` like it is in `news.json`.

Reply:
Thanks Vito, updated this